src/service/packet_router.proto

```diff
@@ -0,0 +1,87 @@
+syntax = "proto3";
+
+package helium.packet_router;
+
+import "region.proto";
+import "data_rate.proto";
+
+message packet_router_route_devaddr_range_v1 {
+  uint64 start = 1;
+  uint64 end = 2;
+}
+
+message packet_router_route_eui_v1 {
+  uint64 app_eui = 1;
+  uint64 dev_eui = 2;
+}
+
+enum packet_router_route_protocol_v1 {
+  gwmp = 0;
+  http = 1;
+  router = 2;
+}
+
+message packet_router_route_v1 {
+  uint64 net_id = 1;
+  repeated packet_router_route_devaddr_range_v1 devaddr_ranges = 2;
+  repeated packet_router_route_eui_v1 euis = 3;
+  bytes lns = 4;
+  packet_router_route_protocol_v1 protocol = 5;
+  uint64 oui = 6;
+}
+
+message packet_router_routes_req_v1 {}
+
+message packet_router_routes_res_v1 {
+  repeated packet_router_route_v1 routes = 1;
+}
+
+message packet_router_packet_up_v1 {
+  bytes payload = 1;
+  uint64 timestamp = 2;
+  uint32 rssi = 3;
+  float frequency_mhz = 4;
+  data_rate datarate = 5;
+  float snr = 6;
+  region region = 7;
+  uint64 hold_time = 8;
+  bytes gateway = 9;
```

Comment:
The `gateway` field may need to include more information than just bytes of its literal key.  For instance, see level of detail in this Rust library:
https://github.com/helium/helium-crypto-rs/blob/main/src/public_key.rs

Namely, which environment? mainnet, testnet; which key type? ECC Compact, ed25519; etc.

While we don't necessarily need to track mainnet/testnet in this field, it should be captured somewhere.